Mage.Tests/src/test/java/org/mage/test/sets/BoosterGenerationTest.java

```diff
@@ -108,35 +108,6 @@ public void testMysteryBooster1() {
         Assert.assertNotNull(booster);
         Assert.assertEquals("Pack contains 15 cards", 15, booster.size());
 
-        Assert.assertTrue("Slot 1 is white", booster.get(0).getColor().isWhite());
-        Assert.assertTrue("Slot 2 is white", booster.get(1).getColor().isWhite());
-
-        // Wretched Gryff is colorless, but stores in blue slots
```

Comment:
Why so many checks/asserts removed?

Reply:
There used to be two redundant tests with slightly different assertions checking the same things. I see no reason the 15 card booster generation needs to be checked separately, because there is already an assert that the standard booster contains 15 cards. I just kept the cleaner code for each case.